app/Foundation/Providers/RouteServiceProvider.php

```diff
@@ -149,8 +170,30 @@ protected function mapOtherwise(Router $router, $routes)
             Timezone::class,
         ];
 
+        if ($applyAlwaysAuthenticate && !$this->isWhiteListedAuthRoute($routes)) {
+            $middleware[] = 'auth.api:true';
```

Comment:
It would be cool if I could use the `::class` class name resolution. But how do I implement this with arguments?

Reply:
I don't think you can, @GrahamCampbell?